diagnostic.md

```diff
@@ -27,7 +27,11 @@ on the body element. Use that variable to increase font-size of `h1`
 elements by 30%.
 
 ```scss
-// your response here
+$h1-font-size:($base-font-size * 1.3);
+
+h1 {
+  font-size: $h1-font-size:;
+}
```

Comment:
Sure, this is ok. But since you're only using `$h1-font-size` with h1 elements, it might just be simpler to write
```
h1 {
  font-size: $base-font-size * 1.3;
}
```